Bio/PDB/PSEA.py

```diff
@@ -17,11 +17,12 @@
 """
 
 import subprocess
+from os.path import exists
```

Comment:
I'm personally not a big fan of imports like this because it's hard for people to know what `exists` refers to. Is it a custom function of ours, or standard library? Unless it's something like `from module.submodule.package import class` (where the call would be super long).

Reply:
Good point, I've just imported os now and done the full call of `os.path.exist()`.